docs/5g-on-helium/mining-5g.mdx

```diff
@@ -165,27 +165,27 @@ following (the below numbers are examples and may not reflect actual deployed 5G
 
 ### Total Rewards Units for Epoch
 
-Calculate the Total Reward Points for a given epoch, by using the Reward Weight for every small cell
-radio type.
+Calculate the Total Reward Points for a given epoch, by using the Reward Weight for each Small Cell
+Radio type.
 
 In this example, the total rewards Points per epoch is 4,000 and the ratios are as follows:
 
 ```bash
-  High-Power Outdoor Count (  150) * 4.0    =>      600
-+            Outdoor Count (  800) * 2.5    =>  + 2,000
-+             Indoor Count (1,400) * 1.0    =>  + 1,400
+  High-Power Outdoor Count (  150) * 4.0    =>      600   =>  0.15
++            Outdoor Count (  800) * 2.5    =>  + 2,000   =>  0.50
++             Indoor Count (1,400) * 1.0    =>  + 1,400   =>  0.35
 -----------------------------------------       --------
-                                                    4,000
+                                                  4,000
 ```
 
 ### Earnings per Small Cell Radio
 
 Each Small Cell Radio is allocated its rewards based on the reward weight of the model.
 
-> Small Cell Reward = Epoch Rewards x Model Reward Weight / Total Reward Points
+> Small Cell Reward = Daily Rewards \* Small Cell Points Ratio / Count of Small Cell Type
```

Comment:
Not sure Small Cell Points Ratio is the best term, just looking at the name it's not clear to me what it is.
Count of Small Cell Type - same here.

Reply:
why not "Reward Weight" since it's referenced above.